tests/MutableArrayTest.php

```diff
@@ -1055,41 +1072,6 @@ public function testDebugReturn(array $array)
         $this->assertTrue($printed === $ma->debug(true));
     }
 
-    public function simpleArrayProvider()
-    {
-        return [
-            // empty array
-            [
-                [],
-            ],
-            // indexed array
-            [
-                [
-                    1 => 'one',
-                    2 => 'two',
-                    3 => 'three',
-                ]
-            ],
-            // assoc array
-            [
-                [
-                    'one' => 1,
-                    'two' => 2,
-                    'three' => 3,
-                ]
-            ],
-            // mixed array
-            [
-                [
-                    1 => 'one',
-                    'two' => 2,
-                    3 => 'three',
-                ]
-
-            ],
-        ];
-    }
-
     public function stringWithSeparatorProvider()
```

Comment:
Could you also move `stringWithSeparatorProvider()` method from both `MutableArrayTest` and `ImmutableArrayTest` classes to the our new `ArrayzyTestTrait` trait, please?
We have equal code in them too.